scenarios.md

```diff
@@ -14,17 +14,17 @@ Notary v2 aims to solve the core issue of trusting content within, and across re
 
 To put Notary v2 in context, the following end-to-end scenario is outlined. The blue elements are the scope of Notary v2, with the other elements providing generic references to other projects or products that demonstrate how Notary v2 may be utilized.
 
-![Notary e2e Scenarios](./media/notary-e2e-scenarios.png)
+![Notary e2e Scenarios](./media/notary-e2e-scenarios.svg)
 
 In a world of consuming public software, we must account for content that's acquired from a public source, copied into a trusted environment, then deployed. In this scenario, the consumer is not re-building or adding additional content. However, they do wish to add attestations to the validity of the content.
 
-1. The Wabbit Networks company builds their netmonitor software.
+1. The Wabbit Networks company builds their `net-monitor` software.
       * As a result of the build, they produce an [OCI Image][oci-image], a Software Bill of Materials (`SBoM`) and to comply with gpl licensing, produce another artifact which contains the source (`src`) to all the gpl licensed projects.  
-      * In addition to the `image`, `SBoM` and `src` artifacts, the build system produces an [OCI Index][oci-index] that encompassed the three artifacts.
-      * Each of the artifacts, and the encompassing `index` are signed with the Notary v2 wabbit-networks key.  
-2. The Wabbit Networks net-monitor index and its signed contents are pushed to a public OCI compliant registry.
+      * In addition to the `image` the `SBoM` and `src` artifacts are created as reference types to the image, creating a graph of artifacts.
```

Comment:
To avoid confusion, I'd just remove the "image" mention here.

```suggestion
      * The `SBoM` and `src` artifacts are created as reference types to the image, creating a graph of artifacts.
```